src/Tribe/Views/V2/Hooks.php

```diff
@@ -60,6 +60,7 @@ protected function add_actions() {
 		add_action( 'tribe_events_pro_shortcode_tribe_events_after_assets', [ $this, 'action_disable_shortcode_assets_v1' ] );
 		add_action( 'updated_option', [ $this, 'action_save_wplang' ], 10, 3 );
 		add_action( 'the_post', [ $this, 'manage_sensitive_info' ] );
+		add_action( 'wp', [ $this, 'mange_single_json_ld' ] );
```

Comment:
I do not know what is "mange", but I would use "print" here.
```suggestion
		add_action( 'wp', [ $this, 'print_single_event_json_ld' ] );
```